tests/ctl/core/test_config.py

```diff
@@ -17,6 +17,13 @@
 }
 
 
+@pytest.mark.unit
+def test_get_config_verbose() -> None:
+    """Simple test to check the 'verbose' code path."""
+    config = get_config(verbose=True)
```

Comment:
@sanders41 the reason I was wondering if I had to do something specific was because this test should make sure that that code path gets called 🤷

Reply:
Is it exiting earlier at the load toml?

Reply:
oh you're right....oof forgot about that, thanks for the sharp catch!